.circleci/config.yml

```diff
@@ -191,8 +191,8 @@ jobs:
       - run:
           name: Install MongoDB
           command: |
-            sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv E162F504A20CDF15827F718D4B7C549A058F8B6B
-            echo "deb [ arch=amd64 ] https://repo.mongodb.org/apt/ubuntu xenial/mongodb-org/4.2 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-4.2.list
+            sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 2930ADAE8CAF5059EE73BB4B58712A2291FA4AD5
```

Comment:
Note, this now runs the machine executor integration tests against the earliest supported MongoDB, which is consistent with the other CI tests.